### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by theyavuzarslan** [availproject/avail#334] [Show and tell]
could you please help me properly?
I appreciate the job you deliver so far at Avail, trying to learn it and create content in Turkish but you dont help me out with the Discord ban. I got banned while I was trying to use faucet. This is the second time for me creating a discussion.
My username is: Swiss Army Knife#1438

**Comment by Ackav** [comment]
Same problem here. Out of nowhere, in the middle of the challenge, banned. 
.nolano on Discord

**Comment by sekasebe** [comment]
Somehow I can not login discord It says "Unable to Accept Invite". Could you please help me about this issue.

**Comment by Leouarz** [comment]
Hello, unfortunately, this place is not the correct one to discuss Discord issues.
